markdown/3.x/en/faq.md

```diff
@@ -100,6 +100,26 @@ If this doesn't seem to solve the issue, please contact us for a custom solution
 
 ---
 
+#### Question:
+Can i contribute?
+
+#### Answer:
+Yes. Although the Web Components codebase is closed, you can still discuss and propose at [GitHub issues](https://github.com/FACT-Finder-Web-Components/ff-web-components/issues).
```

Comment:
Being a bit more specific could inspire readers to read on I think.
Although I'm not entirely sure if this combination of `discuss`, `propose`, `issues` etc isn't a bit quirky.
```suggestion
Yes. Although the Web Components codebase is closed, you can still discuss and propose issues, changes and features at [GitHub issues](https://github.com/FACT-Finder-Web-Components/ff-web-components/issues).
```